packages/web/src/components/notification/Notification/components/EntityLink.tsx

```diff
@@ -35,17 +24,14 @@ export const useGoToEntity = (
 ) => {
   const dispatch = useDispatch()
   const record = useRecord()
-  const isNotificationPanelOpen = useSelector(getNotificationPanelIsOpen)
 
   const handleClick: MouseEventHandler = useCallback(
     (event) => {
       if (!entity) return
       event.stopPropagation()
       event.preventDefault()
       const link = getEntityLink(entity)
-      if (isNotificationPanelOpen) {
-        dispatch(toggleNotificationPanel())
-      }
+      dispatch(closeNotificationPanel())
```

Comment:
do we know why this was guarded before?

Reply:
yes cause toggle could be called multiple times, kinda same issue we were seeing with saliou's flag drama.